python/swe/dockerfiles/create_index.py

```diff
@@ -11,6 +11,8 @@
 from composio.utils.logging import WithLogger
 
 
+print("This is a Test PR")
```

Comment:
Instead of using a plain print statement, consider using a logging framework for better control over output and consistency with other parts of the project. The file already imports 'WithLogger', which suggests logging is used elsewhere in the project. This would provide more flexibility and control over the output, especially in different environments.